js/src/components/google-combo-account-card/connect-google-combo-account-card.scss

```diff
@@ -0,0 +1,11 @@
+.gla-connect-google-combo-account-card {
+
+	.gla-account-card__helper {
+		color: $gray-700;
```

Comment:
This style can be omitted as the same color has been set in `.gla-account-card__helper`.